### PR TITLE
update duration properties with "x-ms-format" : "duration-constant"

### PR DESCRIPTION
**Pull request by mwesigwaguma**
Adding "x-ms-format" : "duration-constant" to duration properties because .Net sdk generation is broken.

Screenshot 2024-01-05 183620 (1)

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Review by a-santamaria — APPROVED** [review]

File: specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2023-12-01-preview/managedcluster.json [hunk 1]
```diff
@@ -992,10 +992,12 @@
       "properties": {
         "healthCheckWaitDuration": {
           "type": "string",
+          "x-ms-format": "duration-constant",
```
Comment (mentat9) [inline specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2023-12-01-preview/managedcluster.json:995]: >"x-ms-format": "duration-constant" [](http://example.com/codeflow?start=10&length=34)

Is this change due to a backend change in validation on these properties, or is it to make the spec match the existing backend? If it's the former, it requires breaking change review and might need a new API version. If it's the latter, then it should be fine.

Comment (mwesigwaguma) [reply specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2023-12-01-preview/managedcluster.json:995]: it's the latter

**Pull request merged**